Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -2590,6 +2613,143 @@ class X86Assembler {
         m_formatter.immediate8((uint8_t)controlBits);
     }
 
+    void vectorFadd(SIMDLane lane, XMMRegisterID left, XMMRegisterID right, XMMRegisterID dest)
```

Comment:
Let's name them with more direct name. This `vectorFadd` abstraction should be done in the MacroAssembler, and X86Assembler should be one-on-one correspondence with the actual X86 instructions.